components/series/stackedBar.js

```diff
@@ -88,7 +50,9 @@
 
         stackedBar.yScale = fc.utilities.property(d3.scale.linear());
 
-        stackedBar.xValueKey = fc.utilities.property('name');
+        stackedBar.layout = fc.utilities.property(d3.layout.stack().offset('zero'));
+
+        stackedBar.stackedData = fc.utilities.property([]);
```

Comment:
See above, I don't think it's possible to expose this at the same time as allow reuse with multiple selections.